File: reservationService/reservationService/src/main/java/com/koroFoods/reservationService/dto/EventoConMesaDto.java
package com.koroFoods.reservationService.dto;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class EventoConMesaDto {
    
    // Datos de EventoMesa
    private Integer idEventoMesa;
    
    // Datos de Evento
    private String nombre;
    private String descripcion;
    private String tematica;
    private LocalDateTime fechaInicio;
    private LocalDateTime fechaFin;
    private String imagen;
    private Boolean activo;
    
    // Datos de Mesa (desde Feign)
    private Integer idMesa;
    private Integer numeroMesa;
    private Integer capacidad;
    private String zona;
}

File: userService/userService/angular/src/app/shared/dto/EventoConMesaDto.ts
export interface EventoConMesaDto {
    // Datos de EventoMesa
    idEventoMesa: number;

    // Datos de Evento
    nombre: string;
    descripcion: string;
    tematica: string;
    fechaInicio: string;
    fechaFin: string;
    imagen: string | null;
    activo: boolean;

    // Datos de Mesa (desde Feign)
    idMesa: number;
    numeroMesa: number;
    capacidad: number;
    zona: string;
}